test/WebJobs.Script.Tests/Binding/ActionResults/RawScriptResultTests.cs

```diff
@@ -83,10 +78,10 @@ public async Task AddsHttpCookies()
             await result.ExecuteResultAsync(context);
             context.HttpContext.Response.Headers.TryGetValue("Set-Cookie", out StringValues cookies);
 
-            Assert.Equal(3, cookies.Count);
+            Assert.Equal(2, cookies.Count);
             Assert.Equal("firstCookie=cookieValue; path=/; samesite=lax", cookies[0]);
+            // TODO: https://github.com/Azure/azure-functions-host/issues/4890
             Assert.Equal("secondCookie=cookieValue2; max-age=20; path=/; httponly", cookies[1]);
-            Assert.Equal("thirdCookie=cookieValue3; path=/; samesite=none", cookies[2]);
         }
```

Comment:
This part:
```javascript
            Assert.Equal(2, cookies.Count);
            Assert.Equal("firstCookie=cookieValue; path=/; samesite=lax", cookies[0]);
            // TODO: https://github.com/Azure/azure-functions-host/issues/4890
            Assert.Equal("secondCookie=cookieValue2; max-age=20; path=/; httponly", cookies[1]);
```
Should be:
```javascript
            Assert.Equal(3, cookies.Count);
            Assert.Equal("firstCookie=cookieValue; path=/; samesite=lax", cookies[0]);
            Assert.Equal("secondCookie=cookieValue2; max-age=20; path=/; httponly", cookies[1]);
            Assert.Equal("thirdCookie=cookieValue3; path=/; samesite=none", cookies[2]);
```

Reply:
This and the above comments around cookie "SameSite" values can also be addressed in a separate PR, and that's ok too. (I was hoping the changes would be ported from dev => v3.x on a merge, but there are definitely complications). Either way, just let me know!

Reply:
I did the merge. Can you check again, please?

Reply:
Looks great, thanks!!